src/qt/forms/debugwindow.ui

```diff
@@ -920,9 +920,30 @@
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <widget class="QTableView" name="peerWidget">
```

Comment:
Upon further investigation I see that Qt Creator validates these types of files and generates these elements when Qt expects them. If they are not present when Qt expects them they generate an innocuous log - So they aren't unnecessary code - these elements are created by Qt when it expects them and is VALID code. I recommend stop considering these auto generated elements as "unnecessary code".

![Screen Shot 2020-12-03 at 3 26 46 PM](https://user-images.githubusercontent.com/152159/101084475-ff92ea80-357b-11eb-9ffe-590f6bd06feb.png)